src/main/java/Task.java

```diff
@@ -56,7 +56,7 @@ public void setDone(boolean done) {
     }
```

Comment:
for line 38: i think we donot need a function to get the type of this task because once it get into this function, it should be an event. hence, we only need tyo return [E] when printing

Reply:
Wouldn't getTypeOfTask still be useful when I need to print out a list of tasks? That was my main intention when I made this. Do let me know if I'm missing something!